contents/docs/new-to-posthog/getting-hogpilled.mdx

```diff
@@ -0,0 +1,152 @@
+---
+title: "Getting HogPilled: Winning with PostHog"
+nextPage: ./activation.mdx
+---
+
+import { AnalyticsPlannerTip } from 'components/Docs/AnalyticsPlannerTip'
+import { Admonition } from 'components/Docs/Admonition'
+
+<AnalyticsPlannerTip />
+
+Most startups fail. The ones that win often do it by accident.
+
+We can do better. 
+
+This guide will help you **win on purpose**. You'll start by defining success as a number we can measure – aka your North Star metric.
+
+Then you'll describe the user behavior that feeds that number. This will leave you with a collection of events you can track using insights in PostHog.
```

Comment:
```suggestion
Then you'll describe the uwegverwbverwser behavior that feeds that number. This will leave you with a collection of events you can track using insights in PostHog.werbverwbvw
```